modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2109,6 +2119,30 @@ public void testDeleteObjectEntry() throws Exception {
 
 		_assertCount(0);
 
+		// Delete object entry when definition is related to a draft object definition
```

Comment:
Hey @thallescezar, just:

```suggestion
		// Delete object entry when its object definition is related to a draft object definition
```

Reply:
@thallescezar after this you can forward to Brian's repo,

Thank you!